doc/source/computation.rst

```diff
@@ -459,6 +459,43 @@ default of the index) in a DataFrame.
    dft
    dft.rolling('2s', on='foo').sum()
 
+.. _stats.rolling_window.endpoints:
+
+Rolling window endpoint inclusion
+~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 0.20.0
+
+The inclusion of the interval endpoints in rolling window calculations can be specified with the ``closed``
+parameter:
+
+  - ``right`` : close right endpoint (default for time-based windows)
```

Comment:
can you make this a csv-table (3 columns: ``closed``, ``description``, ``default``) or something like that